src/main/java/com/blackhearth/blockchain/protocol/message/BasicMessageFactory.java

```diff
@@ -27,6 +28,17 @@
     private final PeerToPeerRepository peerToPeerRepository;
     private final Wallet wallet;
 
+    public BasicMessageFactory(BlockMiner blockMiner,
+                               @Lazy BlockChainNode blockChainNode,
```

Comment:
@Lazy nad polem, @RequiredArgsConstructor zostawiony, powinno zadziałać. Jeśli nie to można zostawić jak jest.

Reply:
Nie działa. Nie poprawiam